src/css/blocks/outlined-circle.css

```diff
@@ -0,0 +1,12 @@
+.outlined-circle {
+  align-items: center;
+  background-color: #d45244;
+  border: 4px solid #ffffff;
+  border-radius: 50%;
+  box-shadow: 0 0 0 5px #d45244;
+  display: inline-flex;
```

Comment:
The inline part is here to center the whole thing in the container. In fact, it should be the responsibility of the container to proper place its elements, not the underlying blocks.